ios/browser/api/ntp_background_images/ntp_sponsored_image.h

```diff
@@ -58,7 +58,7 @@ OBJC_EXPORT
 
 OBJC_EXPORT
 @interface NTPSponsoredImageLogo : NSObject
-@property(readonly) NSURL* imagePath;
+@property(readonly, nullable) NSURL* imagePath;
```

Comment:
@aseren if `imagePath` is null we should not show a blank page, we should just discard the NTT ad.